tests/ats/test_logging_op.py

```diff
@@ -0,0 +1,63 @@
+import logging
+from typing import List
+
+import pykube
+import pytest
+from pytest_helm_charts.clusters import Cluster
+from pytest_helm_charts.k8s.deployment import wait_for_deployments_to_run
+
+logger = logging.getLogger(__name__)
+
+namespace_name = "monitoring"
+deployment_name= "logging-operator"
+
+timeout: int = 560
+
+@pytest.mark.smoke
+def test_api_working(kube_cluster: Cluster) -> None:
+    """
+    Testing apiserver availability.
+    """
+    assert kube_cluster.kube_client is not None
+    assert len(pykube.Node.objects(kube_cluster.kube_client)) >= 1
+
+# scope "module" means this is run only once, for the first test case requesting! It might be tricky
+# if you want to assert this multiple times
+@pytest.fixture(scope="module")
+def ic_deployment(kube_cluster: Cluster) -> List[pykube.Deployment]:
```

Comment:
Could you please add a description on what we are testing here.

Reply:
I added some comments explaining what's being tested for each :)